File: ListaPOOExercicios/Atividade1/Python/Carro.py
class Carro:
    def __init__(self, marca, modelo, ano):
        self.marca = marca
        self.modelo = modelo
        self.ano = ano

    def detalhes_carro(self):
        print(f"Marca: {self.marca}, Modelo: {self.modelo}, Ano: {self.ano}")


carro1 = Carro("Fiat", "Uno", "2020")
carro2 = Carro("Volkswagen", "Gol", "2019")
carro3 = Carro("Chevrolet", "Camaro", "2015")
carro1.detalhes_carro()
carro2.detalhes_carro()
carro3.detalhes_carro()

File: ListaPOOExercicios/Atividade1/Golang/Carro.go
package main

import "fmt"

type Carro struct {
	marca  string
	modelo string
	ano    string
}

func main() {
	var carro1 Carro
	var carro2 Carro
	var carro3 Carro

	carro1 = Carro{
		marca:  "Fiat",
		modelo: "Uno",
		ano:    "2020",
	}
	carro2 = Carro{
		marca:  "Volkswagen",
		modelo: "Gol",
		ano:    "2019",
	}
	carro3 = Carro{
		marca:  "Chevrolet",
		modelo: "Camaro",
		ano:    "2015",
	}

	fmt.Println(carro1, carro2, carro3)
}
